www/js/overlay.js

```diff
@@ -226,17 +233,69 @@ const toggleCast = h(
   h('div.setting-label', `${overlayCast ? 'On' : 'Off'}`),
 );
 
+const toggleLogo = h(
+  'div.input-wrap.logo-toggle',
+  {
+    onclick: evt => {
+      if (overlayCast) {
+        overlayVars.overlayLogo = !overlayVars.overlayLogo;
+        savePrefs();
+        const { overlayLogo } = overlayVars;
+
+        const $logoIcon = evt.currentTarget.querySelector('.cp-icon');
+        $logoIcon.classList.toggle('fa-toggle-on', overlayLogo);
+        $logoIcon.classList.toggle('fa-toggle-off', !overlayLogo);
+      }
+    },
+  },
+  h(
+    'div#logo-btn',
+    h(`i.fa.cp-icon.${overlayVars.overlayLogo ? 'fa-toggle-on' : 'fa-toggle-off'}`),
+  ),
+  h('div.setting-label', 'Logo'),
+);
+
+const toggleAnnouncements = h(
+  'div.input-wrap.announcement-toggle',
+  {
+    onclick: evt => {
+      if (overlayCast) {
+        announcementOverlay = !announcementOverlay;
+        store.setUserPref('app.announcement-overlay', announcementOverlay);
+        ipcRenderer.send('update-settings');
+
+        const $announcementIcon = evt.currentTarget.querySelector('.cp-icon');
+        $announcementIcon.classList.toggle('fa-toggle-on', announcementOverlay);
+        $announcementIcon.classList.toggle('fa-toggle-off', !announcementOverlay);
+      }
+    },
+  },
+  h(
+    'div#announcement-btn',
+    h(`i.fa.cp-icon.${announcementOverlay ? 'fa-toggle-on' : 'fa-toggle-off'}`),
+  ),
+  h('div.setting-label', { style: 'font-size: 9px' }, ['Announce', h('wbr'), 'ment']),
+);
+
+/** Main Control Bar Items */
+controlPanel.append(toggleCast);
+controlPanel.append(toggleLogo);
+controlPanel.append(toggleAnnouncements);
```

Comment:
should we make this panel wider or make the font smaller to make it fit without wrapping 
![image](https://user-images.githubusercontent.com/2982784/69353317-2d3c1880-0c33-11ea-9c66-a3052353d51e.png)

Reply:
+1